### PR TITLE
Feature/rspec test

### PR DESCRIPTION
**Pull request by jpdf00**
Add test suit for all methods in the game_logic.rb

### CODE REVIEW

File: spec/game_logic_spec.rb
Comment (teekaytech) [inline spec/game_logic_spec.rb]: Please use `require_relative` to link files in your directory. 
Using `require_relative` for files in your directory instead of `require` is a ruby convention.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I like the way you structured your test cases. Most importantly, all test cases are passing.
However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Update your Readme.MD file: kindly add instructions on how to run your tests.
- [x] Always test for both positive and negative scenarios. Please go through your test cases and ensure all test cases have their corresponding negative scenarios tested.



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by oloomoses — APPROVED** [review]
# Approved
### Congratulations @jpdf00, :tada: :100: :muscle: 
- This project meets all the minimum requirements and is ready for a merge.
- I wish you the best on your next project.

**Pull request merged**
